packages/expo-apple-authentication/src/AppleAuthentication.types.ts

```diff
@@ -1,12 +1,24 @@
 import { StyleProp, ViewStyle } from 'react-native';
 
-/**
- * @hidden
- */
+// @needsAudit @docsMissing
 export type AppleAuthenticationButtonProps = {
```

Comment:
As the docs suggest we can derive the props from the `View` component, so let's do it, shall we? 😊 
```suggestion
export interface AppleAuthenticationButtonProps extends ViewProps {
```